content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md

```diff
@@ -33,10 +33,8 @@ To read more about {% data variables.product.prodname_dotcom %}'s privacy practi
 
 ## {% data variables.product.prodname_campus_program %} Application Eligibility
 
-* Often times, a campus CTO/CIO, Dean, Department Chair, or Technology Officer signs the terms of the program on behalf of the campus.
+* A school employee with authority to adopt technology for the whole school, such as a CTO/CIO, Dean, Department Chair, or Technology Officer, must sign the terms of the program on behalf of the campus.
 
-* If your school does not issue email addresses, {% data variables.product.prodname_dotcom %} will reach out to your account administrators with an alternative option to allow you to distribute the student developer pack to your students.
+For more information, see the [official {% data variables.product.prodname_campus_program %}](https://education.github.com/schools) page. Or visit the [GitHub Education Resource Guide](https://github.com/github-education-resources/edu-resource-guide?tab=readme-ov-file#github-education-resource-guide).
```

Comment:
```suggestion
For more information, see the [official {% data variables.product.prodname_campus_program %}](https://education.github.com/schools) page or visit the [{% data variables.product.prodname_education %} Resource Guide](https://github.com/github-education-resources/edu-resource-guide?tab=readme-ov-file#github-education-resource-guide).
```

Reply:
These changes all look awesome @sophietheking! Thank you - please go ahead and merge when you can.